main.go

```diff
@@ -162,19 +161,6 @@ func main() {
 	log.SetOutput(io.MultiWriter(os.Stderr, logfile))
 	fileOnlyLogger := log.New(logfile, "", log.LstdFlags)
 
-	log.Print("Starting RHMAP System Dump Tool...")
-	log.Print("Preparing tasks...")
-
-	tasks, err := GetAllTasks(basePath)
-	if err != nil {
-		printError(err)
-		defer os.Exit(1)
-	}
-	if len(tasks) == 0 {
-		log.Print("No tasks found to execute.")
```

Comment:
@philbrookes now we don't handle this case anymore.

No tasks really mean we've made a programming mistake, don't you think?